pkgs/applications/misc/pdfslicer/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, gettext
+, intltool
+, pkg-config
+, wrapGAppsHook
+, gtkmm3
+, libuuid
+, poppler
+, qpdf
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pdfslicer";
+  version = "1.8.8";
+
+  src = fetchFromGitHub {
+    owner = "junrrein";
+    repo = "pdfslicer";
+    rev = "v${version}";
+    fetchSubmodules = true;
```

Comment:
Can we unvendor the dependencies? The vendored ones are quite old.

Reply:
I want to try doing that, but I won't have time for a while.

Reply:
I tried this over at https://github.com/NixOS/nixpkgs/pull/116617.